include/crypto_cfg.h

```diff
@@ -36,4 +36,6 @@
 #define CRYPTO_FUNCTION __attribute__((section("crypto_functions")))
 #define CRYPTO_DATA __attribute__((section("crypto_data")))
 
+#define CRYPTO_CONST_DATA __attribute__((section(".data")))
```

Comment:
This is a bit dirty, this improve the performance by avoiding access to flash. For standards curves structures. I am open to suggestion.

Reply:
By the way, this is not mandatory.

Reply:
It is really confusing for the reader (without comments), and not always true (performance increase). For example with the secure boot, if non XiP mode is used, `.rodata` section is also copied in RAM.

I'm not sure we should enforce this kind of optimization in a SDK, with potential adverse behavior (example: flash with a data cache). It also means that more RAM is used even if none or a little data is actually used at run time.

 * It could either be disabled by default and enabled with a specific macro, or
 * it could be defined as specific section and the linker script be used to define the proper location depending on the actual target, such as `crypto_rodata`.

Reply:
Ok I change that. And added note in readme instead.

Reply:
`This avoid very Flash access` -> This avoid *many* Flash access ?

Reply:
I think the comments (readme) should document that this is useful IF application runs from flash, which is only an option among several possibilities. Otherwise, it may uselessly duplicate the section in RAM.

Reply:
My bad, half erased sentence.

Reply:
I added a clarification on the suggested optimization

Reply:
I added a note on the potential security issue of pacing constant data in a writable section.